femalifing_com/docker-compose.yml

```
@@ -0,0 +1,19 @@
+version: '3'
+
+# note: the name of the domain which this container serves is "mesuochi.com",
+#       though the container's name is "femalifing_com".
+#       This name shake is for reducing the mental burden
+#       when typing this container's name, or looking this container's name, and so on.
+#
+#       yes don't forget that humans maybe dont like mesuochi
+#       (maybe some people (like me) do)
+
+services:
+  femalifing_com:
+    image: docker.pkg.github.com/approvers/mesuochi.com/mesuochi.com:0.1.2
+    container_name: femalifing_com
+    restart: unless-stopped
+    ports:
+      - 21001:80
```

```suggestion
      - 14003:80
```

`140XX` を使用してください (他コンテナの設定参照のこと)

---

1176a4b で修正しました:bow: